src/SearchParameters/RefinementList.js

```diff
@@ -118,19 +121,19 @@ var lib = {
    * @param {string} refinementValue value of the filter/refinement
    * @return {boolean}
    */
-  isRefined : function isRefined( refinementList, attribute, refinementValue ) {
-    var containsRefinements = refinementList[ attribute ] &&
-                              refinementList[ attribute ].length > 0;
+  isRefined: function isRefined(refinementList, attribute, refinementValue) {
+    var containsRefinements = refinementList[attribute] &&
+      refinementList[attribute].length > 0
 
-    if( isUndefined( refinementValue ) ) {
-      return containsRefinements;
+    if (isUndefined(refinementValue)) {
+      return containsRefinements
     }
 
-    var refinementValueAsString = "" + refinementValue;
+    var refinementValueAsString = '' + refinementValue
 
     return containsRefinements &&
-           refinementList[ attribute ].indexOf( refinementValueAsString ) !== -1;
+      refinementList[attribute].indexOf(refinementValueAsString) !== -1
```

Comment:
Yes that's http://eslint.org/docs/rules/array-bracket-spacing and http://eslint.org/docs/rules/space-in-parens